doc/examples/ex41/ex41.sh

```diff
@@ -5,9 +5,9 @@
 # GMT modules:  set, coast, legend, plot, makecpt
 #
 gmt begin ex41
-	gmt set FONT_ANNOT_PRIMARY 12p FONT_LABEL 12p
+	gmt set MAP_AUTO_SCALE off FONT_ANNOT_PRIMARY 12p,Helvetica FONT_LABEL 12p,Helvetica
```

Comment:
`MAP_AUTO_SCALE` was removed.
```suggestion
	gmt set FONT_ANNOT_PRIMARY 12p,Helvetica FONT_LABEL 12p,Helvetica
```